### DISCUSSION TITLE
Why are jobs getting stuck with no progress

### DISCUSSION THREAD

**Discussion by cyperbg** [ericaltendorf/plotman#494] [Q&A]
So I ran 4 plots on a 1TB SSD, which led to lack of space. I then killed 2, so only 2 were left, but they seems to be stuck.

64405ca7   32     /media/cyper/evo_1tb   /media/cyper/Farm1   12:12     3:3   186G   155269    STP   4.0G   6:12   0:20   0:11
573f41b8   32     /media/cyper/evo_1tb   /media/cyper/Farm1   13:43     3:5   148G   144454    STP   4.0G   7:27   0:22   0:36

I manually cleaned the ssd, so there is plenty of free space + suspend then resume them.
The log file just stays at:
Bucket 83 uniform sort. Ram: 1.627GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.

Have I lost them or they can still recover?

**Comment by altendky** [comment]
FWIW, with successful staggering I have been running 4x on 1tb even with dst set the same as tmp without any trouble except when archiving hangs up and fails (something with rsync and my network).

What you show there shows the jobs suspended, that's what `STP` is.  If they aren't suspended at this point and still don't make any progress then that's a Chia question.  Not to push you out, but I don't offhand have the expertise there.  The #support channel on the Chia keybase would be a better bet in that case.

Also, plotman isn't presently great at cleaning up _all_ the tmp files for processes it kills so you may need to manually clean some up as well.  One of the many bits we need to work on.
